### PR TITLE
Remove custom token

### PR DESCRIPTION
**Pull request by angarita-dev**
## Changelog

- Allow Functionality to remove custom token. #579 
- Requires to merge & publish https://github.com/Psychedelic/plug-controller/pull/206

### Type of changes included:

- [ ] Components
- [x] Business Logic
- [ ] Bug fixes
- [ ] Styling
- [x] Code Refactor

### Screenshots/Gifs:

https://user-images.githubusercontent.com/44899916/188564668-9c1a6be6-862a-4cf5-88b2-1333983129e8.mov

### Notes:

*Place special notes here*

### Tested on:

- [x] Chrome
- [ ] Firefox
- [ ] Safari

### CODE REVIEW

File: source/Background/Keyring/index.js [hunk 1]
```diff
@@ -136,6 +136,7 @@ export const getKeyringErrorMessage = (type) => ({
   [HANDLER_TYPES.REMOVE_NETWORK]: 'removing the network',
   [HANDLER_TYPES.SET_CURRENT_NETWORK]: 'setting the current network',
   [HANDLER_TYPES.GET_CURRENT_NETWORK]: 'getting the current network',
+  [HANDLER_TYPES.REMOVE_ASSET]: 'removing a token',
```
Comment (ignaciosantise) [inline source/Background/Keyring/index.js]: adding a token is named `ADD_CUSTOM_TOKEN`. I think we should name this one `REMOVE_TOKEN` or `REMOVE_CUSTOM_TOKEN`. What u think?

Comment (angarita-dev) [reply source/Background/Keyring/index.js]: This makes perfect sense 🙏. Just updated it

File: source/components/AssetItem/index.jsx [hunk 2]
```diff
@@ -8,27 +8,88 @@ import clsx from 'clsx';
...
+      <ActionDialog
+        open={openDelete}
+        title={t('removeToken.action')}
+        content={(
+          <Typography>
+            {t('removeToken.mainText')}
+            <b>{symbol}</b>
+            {t('removeToken.mainTextContinue')}
+            <br />
+            <br />
```
Comment (tomiir) [inline source/components/AssetItem/index.jsx:84]: why use these? isnt a margin better?

**Review by tomiir — APPROVED** [review]
You could def split that `Typography` into smaller components and style them properly, else seems ok

**Pull request merged**